integration_tests/src/expects.rs

```diff
@@ -45,6 +46,13 @@ impl Expect {
         sector_type: RegisteredSealProof,
         compute_cid: bool,
     ) -> ExpectInvocation {
+        let activated_events: Vec<EmittedEvent> = deals
+            .iter()
+            .map(|deal_id| {
+                build_market_event("deal-activated", STORAGE_MARKET_ACTOR_ID, deal_id.to_owned())
```

Comment:
For all built-in tests, the emitter is always the built-in market, so you can probably remove that parameter.

Reply:
Done.